README.md

```diff
@@ -453,6 +452,100 @@ It will then locate the event service and traverse the members of the event dest
+    info                Displays information about the virtual media for a
+                        system
+    insert              Inserts virtual media for a system
+    eject               Ejects virtual media from a system
+
+required arguments:
+  --user USER, -u USER  The user name for authentication
+  --password PASSWORD, -p PASSWORD
+                        The password for authentication
+  --rhost RHOST, -r RHOST
+                        The address of the Redfish service (with scheme)
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --system SYSTEM, -s SYSTEM
+                        The ID of the system perform the operation
+```
+
+#### Info
+
+```
+usage: rf_virtual_media.py info [-h]
+
+optional arguments:
+  -h, --help  show this help message and exit
+```
+
+Example: `rf_virtual_media.py -u root -p root -r https://192.168.1.100 info`
+
+The tool will log into the service specified by the *rhost* argument using the credentials provided by the *user* and *password* arguments.
+It will then locate the system specified by the *system* argument, find its virtual media collection, and display the virtual media instances.
+
+
+#### Insert
+
+```
+usage: rf_virtual_media.py insert [-h] --image IMAGE [--id ID] [--notinserted]
+                                  [--writable]
+                                  [--mediatypes MEDIATYPES [MEDIATYPES ...]]
+
+required arguments:
+  --image IMAGE, -image IMAGE
+                        The URI of the image to insert
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --id ID, -i ID        The identifier of the virtual media instance to insert
+  --notinserted, -notinserted
+                        Indicates if the media is to be marked as not inserted
+                        for the system
+  --writable, -writable
+                        Indicates if the media is to be marked as writable for
+                        the system
+  --mediatypes MEDIATYPES [MEDIATYPES ...], -mt MEDIATYPES [MEDIATYPES ...]
+                        A list of acceptable media types for the virtual media
+``` 
+
+Example: `rf_virtual_media.py -u root -p root -r https://192.168.1.100 insert -image http://somefileserver/my_media.iso`
+
+The tool will log into the service specified by the *rhost* argument using the credentials provided by the *user* and *password* arguments.
+It will then locate the system specified by the *system* argument, find its virtual media collection.
+It will then iterate through the virtual media collection, and insert the virtual media specified by the *image* argument in an appropriate slot.
+
+
+#### Eject
+
+```
+usage: rf_virtual_media.py eject [-h] --id ID
+
+required arguments
+  --id ID, -i ID  The identifier of the virtual media instance to eject
+
+optional arguments:
+  -h, --help      show this help message and exit
+```
+
+Example: `rf_event_service.py -u root -p root -r https://192.168.1.100 unsubscribe -id 1`
```

Comment:
```suggestion
Example: `rf_event_service.py -u root -p root -r https://192.168.1.100 eject -id 1`
```

Reply:
Ick, I forgot to update that; there are some other things wrong with it too. Thanks for pointing that out.